lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -0,0 +1,240 @@
+        background-image: url(${fullpageScreenshotUrl})
+      }`;
+    containerEl.appendChild(fullPageScreenshotStyleTag);
+  }
+
+  /**
+   * @param {DOM} dom
+   * @param {ParentNode} templateContext
+   * @param {LH.Artifacts.FullPageScreenshot} fullPageScreenshot
+   */
+  static installOverlayFeature(dom, templateContext, fullPageScreenshot) {
+    ElementScreenshotRenderer._installBackgroundImageStyle(dom, fullPageScreenshot);
+
+    const reportEl = dom.find('.lh-report', dom.document());
+    const renderContainerSizeInDisplayCoords = {
+      width: dom.document().documentElement.clientWidth - reportEl.getBoundingClientRect().left,
+      height: dom.document().documentElement.clientHeight - reportEl.getBoundingClientRect().top,
+    };
+    for (const el of dom.document().querySelectorAll('.lh-element-screenshot')) {
+      el.addEventListener('click', () => {
+        const overlay = dom.createElement('div');
+        overlay.classList.add('lh-element-screenshot__overlay');
+        const elementRectInScreenshotCoords = {
+          width: Number(el.getAttribute('rectWidth')),
+          height: Number(el.getAttribute('rectHeight')),
+          left: Number(el.getAttribute('rectLeft')),
+          right: Number(el.getAttribute('rectLeft')) + Number(el.getAttribute('rectWidth')),
+          top: Number(el.getAttribute('rectTop')),
+          bottom: Number(el.getAttribute('rectTop')) + Number(el.getAttribute('rectHeight')),
+        };
+        overlay.appendChild(ElementScreenshotRenderer.render(
+          dom,
+          templateContext,
+          fullPageScreenshot,
+          elementRectInScreenshotCoords,
+          renderContainerSizeInDisplayCoords
+        ));
+        overlay.addEventListener('click', () => {
+          overlay.remove();
+        });
+
+        reportEl.appendChild(overlay);
+      });
+    }
+  }
+
+  /**
+   * @param {LH.Artifacts.Rect} elementRectInScreenshotCoords
+   * @param {Size} renderContainerSizeInDisplayCoords
+   * @return {number}
+   */
+  static _computeZoomFactor(elementRectInScreenshotCoords, renderContainerSizeInDisplayCoords) {
+    const targetClipToViewportRatio = 0.75;
+    const zoomRatioXY = {
+      x: renderContainerSizeInDisplayCoords.width / elementRectInScreenshotCoords.width,
+      y: renderContainerSizeInDisplayCoords.height / elementRectInScreenshotCoords.height,
+    };
+    const zoomFactor = targetClipToViewportRatio * Math.min(zoomRatioXY.x, zoomRatioXY.y);
+    return Math.min(1, zoomFactor);
+  }
+
+  /**
+   * @param {DOM} dom
+   * @param {ParentNode} templateContext
+   * @param {LH.Artifacts.FullPageScreenshot} fullPageScreenshot
+   * @param {LH.Artifacts.Rect} elementRectInScreenshotCoords Region of screenshot to highlight.
+   * @param {Size} renderContainerSizeInDisplayCoords
+   * @return {Element}
+   */
+  static render(dom, templateContext, fullPageScreenshot, elementRectInScreenshotCoords,
+      renderContainerSizeInDisplayCoords) {
+    //
```

Comment:
this is the second time I see this so I'm guessing there might be a method to it that I'm missing :)

Reply:
It's to create space between the parameter list and the body of the function.